src/components/combos/ComboList.js

```diff
@@ -18,11 +22,14 @@ const ComboList = ({ sortOpt, scrollPosition, setScrollPosition }) => {
   });
 
   // combos query with react hooks -- means it refreshes automaticly
-  const combos = useLiveQuery(() => db.getAllCombos().then(c => c.sort(sortingSchemes[sortOpt].sortFunc)), [sortOpt]);
+  const combos = useLiveQuery(() => db.getAllCombos().then(c => c.sort(comboSortingSchemes[sortOpt].sortFunc)), [sortOpt]);
```

Comment:
Not really your code so feel free to disregard. But while we are at it: It would be more readable to move the functionality into a separate function. Like so:

```js
const combosFromDBSorted = async () => {
  const allCombos = await db.getAllCombos();
  return allCombos.sort(comboSortingSchemes[sortOpt].sortFunc);
}

// combos query with react hooks -- means it refreshes automatically
const combos = useLiveQuery(() => combosFromDBSorted(), [sortOpt]);
```